src/ExplorerDetector.cpp

```diff
@@ -25,41 +25,50 @@ G4VPhysicalVolume* ExplorerDetector::Construct( std::string Name, G4LogicalVolum
   G4bool isotopes = false;
 
   G4Material* crystal = nullptr;
-  if ( Material == "LSO" )
-  {
-    G4Element* O  = nistManager->FindOrBuildElement( "O" , isotopes );
-    G4Element* Si = nistManager->FindOrBuildElement( "Si", isotopes );
-    G4Element* Lu = nistManager->FindOrBuildElement( "Lu", isotopes );
-
-    G4Material* LSO = new G4Material( "Lu2SiO5", 7.4*g/cm3, 3 );
-    LSO->AddElement( Lu, 2 );
-    LSO->AddElement( Si, 1 );
-    LSO->AddElement( O , 5 );
-
-    crystal = LSO;
-  }
-  else if ( Material == "LYSO" || Material == "" ) //default
-  {
-    // exact composition for EXPLORER from https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6354919/
-    G4Element* O  = nistManager->FindOrBuildElement( "O" , isotopes );
-    G4Element* Si = nistManager->FindOrBuildElement( "Si", isotopes );
-    G4Element* Lu = nistManager->FindOrBuildElement( "Lu", isotopes );
-    G4Element* Y  = nistManager->FindOrBuildElement( "Y" , isotopes );
-
-    G4Material* LYSO = new G4Material( "LYSO", 7.1*g/cm3, 4 );
-    LYSO->AddElement( Lu, 71.447 * perCent );
-    LYSO->AddElement( Y,  4.034  * perCent );
-    LYSO->AddElement( Si, 6.371  * perCent );
-    LYSO->AddElement( O,  18.148 * perCent );
 
+  G4Element* O  = nistManager->FindOrBuildElement( "O" , isotopes );
```

Comment:
In future, this common material code should probably live in its own function rather than being duplicated between Explorer and Siemens (and other) detectors.